test/e2e/dns.go

```diff
@@ -41,6 +41,8 @@ var dnsServiceLabelSelector = labels.Set{
 	"kubernetes.io/cluster-service": "true",
 }.AsSelector()
 
+var ClusterDNSVerifier = verifyDNSPodIsRunning
```

Comment:
Yes, will make a comment that this is done to expose external testing.

On Tue, Apr 12, 2016 at 12:33 AM, Erick Fejta <notifications@github.com>
wrote:

> In test/e2e/dns.go
> <https://github.com/kubernetes/kubernetes/pull/24128#discussion_r59319573>
> :
>
> > @@ -41,6 +41,8 @@ var dnsServiceLabelSelector = labels.Set{
> >  	"kubernetes.io/cluster-service": "true",
> >  }.AsSelector()
> >
> > +var ClusterDNSVerifier = verifyDNSPodIsRunning
>
> Does this mean we won't be able to change this var without breaking people
> outside of this repo? If so will you make a comment to this effect?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24128/files/e14c4bc724d696d726cc335914a24eb61120aaa2#r59319573>
>